packages/core/src/components/misc/CredentialCard10.tsx

```diff
@@ -153,11 +154,33 @@ const CredentialCard10: React.FC<CredentialCard10Props> = ({ credential, style =
       language: i18n.language,
     }
     bundleResolver.resolveAllBundles(params).then((bundle) => {
+      // Use cached schema name as fallback if no specific overlay name is found
+      const cachedSchemaName = getSchemaName(credential)
+      // Check if OCA name is meaningful (not just "Credential" or empty)
+      const ocaName = bundle.metaOverlay?.name
+      const hasMeaningfulOcaName = ocaName && ocaName !== 'Credential' && ocaName.trim() !== ''
```

Comment:
Could this be a custom use case to your current testing scenario? For example why not "Le justificatif" in French or "Der Ausweis" in German rather than "Credential" in English? Where I'm going here is are we assuming the English word is always used here? I though OCA always had the built in concept of i8n - no?